assets/src/application/ui/layout/entityActionsMenu/EntityActionsMenu.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import classNames from 'classnames';
+
+export enum EntityActionsMenuLayout {
+	Horizontal = '--horizontal',
+	Vertical = '--vertical',
+}
+
+export interface EntityActionsMenuProps {
+	className: string;
+	layout: EntityActionsMenuLayout;
+	menuItems: Array<React.ReactNode>;
+	[key: string]: any;
+}
+
+const EntityActionsMenu: React.FC<EntityActionsMenuProps> = ({
+	className,
+	layout = EntityActionsMenuLayout.Horizontal,
+	menuItems,
+	...menuProps
+}) => {
+	className = className ? `${className} ee-entity-actions-menu` : 'ee-entity-actions-menu';
+	const classes = classNames({
+		[className]: className,
+		'ee-entity-actions-menu': true,
+		'ee-entity-actions-menu--horizontal': layout === EntityActionsMenuLayout.Horizontal,
+		'ee-entity-actions-menu--vertical': layout === EntityActionsMenuLayout.Vertical,
+	});
```

Comment:
me being nitpicky:
```
const horizontal = EntityActionsMenuLayout.Horizontal;
const vertical = EntityActionsMenuLayout.Vertical;

const classes = classNames({
		[className]: className,
		'ee-entity-actions-menu': true,
		`ee-entity-actions-menu${horizontal}`: layout === horizontal,
		`ee-entity-actions-menu${vertical}`: layout === vertical,
	});
```
very optional

Reply:
oh right I meant to do that actually, which is why the constant values includes `--`

Reply:
decided to change the enum values to 

```
	Horizontal = 'horizontal',
	Vertical = 'vertical',
```

which likely makes things less error prone since consuming code simply has to add `layout='vertical'` to a component without having to add `--` or import the constants